index.test-d.ts

```diff
@@ -22,8 +34,12 @@ intoStream(Promise.resolve(new Uint8Array(Buffer.from('unicorn').buffer))).pipe(
 	process.stdout
 );
 
+intoStream(asyncGeneratorFrom(unicornArray)()).pipe(process.stdout);
+intoStream(asyncIterableFrom(unicornArray)).pipe(process.stdout);
+intoStream(Promise.resolve(asyncIterableFrom(unicornArray))).pipe(process.stdout);
+
 const object = {foo: true};
-const objectArray = new Set([object, {bar: true}]);
+const objectArray = [object, {bar: true}];
```

Comment:
Should I open a separate PR for this fix?

Reply:
Nah, it's fine.